File: API/Client/GetPricesByOfferIdsResponse.js
goog.provide('API.Client.GetPricesByOfferIdsResponse');

/**
 * Ответ на запрос списка цен.
 * @record
 */
API.Client.GetPricesByOfferIdsResponse = function() {}

/**
 * @type {!API.Client.ApiResponseStatusType}
 * @export
 */
API.Client.GetPricesByOfferIdsResponse.prototype.status;

/**
 * @type {!API.Client.OfferPriceByOfferIdsListResponseDTO}
 * @export
 */
API.Client.GetPricesByOfferIdsResponse.prototype.result;

File: src/main/scala/org/openapitools/client/api/GetPricesByOfferIdsResponse.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import GetPricesByOfferIdsResponse._

case class GetPricesByOfferIdsResponse (
  status: Option[ApiResponseStatusType],
result: Option[OfferPriceByOfferIdsListResponseDTO])

object GetPricesByOfferIdsResponse {
  import DateTimeCodecs._

  implicit val GetPricesByOfferIdsResponseCodecJson: CodecJson[GetPricesByOfferIdsResponse] = CodecJson.derive[GetPricesByOfferIdsResponse]
  implicit val GetPricesByOfferIdsResponseDecoder: EntityDecoder[GetPricesByOfferIdsResponse] = jsonOf[GetPricesByOfferIdsResponse]
  implicit val GetPricesByOfferIdsResponseEncoder: EntityEncoder[GetPricesByOfferIdsResponse] = jsonEncoderOf[GetPricesByOfferIdsResponse]
}
